packages/lisk-transactions/src/3_cast_votes.js

```diff
@@ -21,9 +21,13 @@ import {
 	wrapTransactionCreator,
 } from './utils';
 
-const castVotes = ({ passphrase, votes = [], unvotes = [] }) => {
+const validateInputs = ({ votes, unvotes }) => {
 	validatePublicKeys([...votes, ...unvotes]);
+};
 
+const castVotes = inputs => {
+	const { passphrase, votes = [], unvotes = [] } = inputs;
+	validateInputs({ votes, unvotes });
```

Comment:
Prefer a consistent approach here: `validateInputs(inputs)` followed by destructuring. (You can provide default values in the `validateInputs` function.)